openapi/openapi.yaml

```diff
@@ -206,6 +217,32 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/APIError"
+    patch:
+      summary: Updates a system by id
+      operationId: systemsPatchById
```

Comment:
It seems that `systemsUpdateById` would be more appropriate, as the word `Patch` is not used anywhere else?